addons/event/report/event_event_templates.xml

```diff
@@ -154,9 +158,14 @@
                     <div class="o_event_full_page_ticket_barcode">
                     
                         <div class="o_event_full_page_ticket_barcode_container px-2">
-                            <div class="pb-3"><img t-attf-src="/report/barcode/QR/{{attendee.barcode if attendee else '12345678901234567890'}}?&amp;width=116&amp;height=116&amp;quiet=0" alt="QR Code"/></div>
-                            <t t-if="env['ir.config_parameter'].sudo().get_param('event.use_event_barcode') == 'True'">
-                                <img class="o_event_barcode" t-attf-src="/report/barcode/?barcode_type=Code128&amp;value={{attendee.barcode if attendee else '12345678901234567890'}}&amp;width=168&amp;height=84&amp;humanreadable=1&amp;quiet=0" alt="Barcode"/>
+                            <t t-if="attendee"> 
```

Comment:
Same as before, we still want to show it in the reports if there's no attendee. Let's use the same condition